doc/sphinxdoc/research_papers.md

```diff
@@ -80,7 +80,11 @@ focused on weblogs. In 7th International Conference on Music Information Retriev
 
 ## Emotion detection
 
+<<<<<<< HEAD
```

Comment:
Some merge conflict here.

Reply:
Maybe it was a local issue? This PR doesn't modify that file and below I can see `This branch has no conflicts with the base branch`